UserDocs.md

```diff
@@ -17,242 +17,228 @@ Install T4:
 $ pip install git+https://github.com/quiltdata/t4.git#subdirectory=ocean
 ```
 
-## API reference
+## User guide
+### Creating a package
 
-> T4 expects S3 paths of the form `BUCKET_NAME/path/to/dir/data.csv`.
+To create a new in-memory package you initialize a new Package object and then call mutators on it until you have the Package you want. Note that Packages are immutable, so set, update, and delete all return new Package instances. For example:
```

Comment:
It's a bit confusing to talk about calling mutators then explaining that the package is immutable. Maybe just call out set and update by name instead of calling them mutators.

Reply:
I've omitted the comment about immutability and tweaked the wording otherwise. Still feeling this out. I don't watn to reference `set` and `update` by name because there's also `capture` and `delete`, which starts to feel like a lot of different methods to call out in text.